assets/src/scripts/wdfn-home/wdfn-param-filters/index.js

```diff
@@ -122,10 +122,32 @@ const WDFNParamFilters = (node, store) => {
     mapContainerEl.dataset.loadingState = loadingState;
   };
 
+  const toggleSearchEnabled = (_, filters) => {
+    let activeFilters = 0;
```

Comment:
To keep users from submitting huge queries, disable the search button when fewer than two filters are applied. This function checks whether at least one value has been specified for at least two filter types (site type, time period, parameter, or geographic)